src/guidellm/executor/profile_generator.py

```diff
@@ -61,25 +61,24 @@ def next_profile(
         pass
 
 
-@ProfileGenerator.register_generator(ProfileGenerationModes.SINGLE)
-class SingleProfileGenerator(ProfileGenerator):
-    def __init__(self, rate: float, rate_type: str, **kwargs):
-        super().__init__(ProfileGenerationModes.SINGLE)
-        self._rate = rate
+@ProfileGenerator.register_generator(ProfileGenerationModes.MULTI)
+class MultiProfileGenerator(ProfileGenerator):
```

Comment:
Not sure if I'm a fan of the Multi naming for this, specifically for two reasons:
1. The Sweep profile runs multiple as well, so it doesn't necessarily differentiate it from that functionality in name
2. This runs single as well as multi, so ideally would like a name to encapsulate this as well

Maybe something along the lines of FixedRateProfileGenerator? Let me know if you have any further ideas towards that line of thinking

Reply:
Yeah I was not a fan of calling it multi as well, but I didn't have a good sense of a better name. Something like FixedRate or CustomRate seems good. Custom rate focuses too much on the fact that a user can pass values in which feels weird, fixed rate feels closer to the mark given that a list is being passed in, seems obvious what its purpose would be